fix: disable header field page action with tooltip (#3016)
SAP/open-ux-tools: .changeset/yellow-rocks-perform.md

---
'@sap-ux-private/preview-middleware-client': patch
'@sap-ux/preview-middleware': patch
---

> fix: disable header field page action when show header content are set to false

Reviewer: ```suggestion
fix: disable "Add Header Field" Quick Action when `showHeaderContent` is set to `false` for `ObjectPageLayout`
```
